static/src/javascripts/projects/facia/modules/onwards/weather.js

```diff
@@ -136,16 +123,7 @@ define([
             return this.getWeatherData(config.page.forecastsapiurl + '/' + location.id + '.json?_edition=' + config.page.edition.toLowerCase())
                 .then(function (response) {
                     this.renderForecast(response);
-                }.bind(this))
-                .fail(function (err, msg) {
-                    var statusText = (err && err.statusText) || '';
-                    var statusCode = (err && err.status) || '';
```

Comment:
`err` might be an `Error` object with custom properties set. In which case, we'll have a `message` and `stack` prop.